Makefile

```diff
@@ -74,6 +74,10 @@ unit-test:
 		--succinct \
 		-ldflags="$(ldflags)" \
 		-r provider
+	ginkgo run \
+		--succinct \
+		-ldflags="$(ldflags)" \
+		-r internal/...
```

Comment:
```suggestion
		-r provider internal/...
```

Reply:
I tried this, it didn't work

Reply:
maybe only with a newer ginkgo I guess

Reply:
Its seems that I had a mistake before, now it works :-)
Fixed